images/disks-images-provider/entrypoint.sh

```diff
@@ -59,6 +60,28 @@ if [ ${SELINUX_TAG:0:1} != "?" ]; then
     chcon -Rt svirt_sandbox_file_t /hostImages
 fi
 
+# Create a 4Gi blank disk image
+dd if=/dev/zero of=/local-storage/hp_file.img bs=4k count=1024k
+ls -al /local-storage/hp_file.img
+# Put LOOP_DEVICE_HP in /etc/bashrc in order to detach this loop device when the pod stopped.
+LOOP_DEVICE_HP=$(chroot /host losetup --verbose --find --show /mnt/local-storage/hp_file.img)
+echo $LOOP_DEVICE_HP
```

Comment:
Leftover debug print?

Reply:
Yes and no, I can remove it, but I think it would be useful for people to see what loopback device was created.

Reply:
If you leave it I'd suggest making it a bit more verbose so people know what it's about!

Reply:
I am not that passionate about it, I am removing it.

Reply:
if people really want to know, you can look in /etc/bashrc to find out.